src/curve/stable/StablePair.sol

```diff
@@ -44,17 +42,13 @@ contract StablePair is ReservoirPair {
 
     AmplificationData public ampData;
 
-    uint256 private _locked = 1;
-
     // We need the 2 variables below to calculate the growth in liquidity between
```

Comment:
Good spot, weird we didn't notice - guess we don't have solhint running?

Reply:
so I just tried, and solhint doesn't catch this. It's like the unused imports, solhint doesn't catch it currently. 

Also I just learned that there's a solhint rule that recommends not more than having 15 state variables

Reply:
Interesting, ty for checking